inventory/hosts

```diff
@@ -108,11 +108,12 @@ uk-staging2
 [us-prod]
 openfoodnetwork.net ansible_host=52.3.171.208
 
-[us-staging]
+[us-migrate]
+migrate.openfoodnetwork.net ansible_host=64.227.102.29
 
 [us:children]
 us-prod
-us-staging
```

Comment:
it doesn't exist anymore? just curious.

Reply:
I don't think it ever existed? I made one very briefly but it wasn't needed.